config/StoredProcedures/AddProductSize

```diff
@@ -1,16 +0,0 @@
-# Used to add an entry to the SizeMap table
-# usage: CALL AddProductSize(101, "BigBundle", 200);
-
-USE imp_db_dev;
-DROP PROCEDURE IF EXISTS AddProductSize;
-
-DELIMITER $$
-CREATE PROCEDURE AddProductSize
-(IN _ProductID int unsigned, IN _Name varchar(50), IN _Size int unsigned)
-BEGIN
-
-INSERT INTO SizeMap
-VALUES (null, _ProductID, _Name, _Size);
-
-END $$
-DELIMITER ;
```

Comment:
We had two AddProductSize files. I removed the other and renamed this file to have the other's name, which caused  the diff view below. This is now the single existing AddProductSize file, but renamed to AddProductSize.sql.